test/e2e/info_test.go

```diff
@@ -49,4 +53,32 @@ var _ = Describe("Podman Info", func() {
 		Expect(session.ExitCode()).To(Equal(0))
 		Expect(session.IsJSONOutputValid()).To(BeTrue())
 	})
+	It("podman info rootless storage path", func() {
+		if !rootless.IsRootless() {
+			Skip("test rootless_storage_path for rootless")
+		}
+		SkipIfRemote()
+		configPath := "/etc/containers/storage.conf"
+		desPath := "/etc/containers/storage.conf.test"
+		cmd := exec.Command("sudo", "cp", configPath, desPath)
+		out, err := cmd.CombinedOutput()
+		fmt.Println(string(out))
+		Expect(err).To(BeNil())
+
+		rootStoragePath := `"/tmp/$HOME/$USER/$UID"`
```

Comment:
Why this complicated (and unreliable) value? It's not clear to me, from the commit message nor the code, what the point of this test is. Are you testing that podman can perform environment variable interpolation in `rootless_storage_path`, or simply that it will recognize an override? If the latter, it would be much better to define `rootStoragePath := /some/randomly/generated/string` and then avoid the duplication in line 73 (setting `expect`).

Reply:
it's testing both variable interpolation in rootless_storage_path and that rootless_storage_path override the GraphRoot value in podman.

Reply:
Ah, okay, that wasn't clear from the commit message. In that case, perhaps a good way to test both would be:
```
    path = "/tmp/$RANDOM_STRING"
    ...write that to $HOME/.config/containers/storage.conf (yes, *BEFORE* setting the string)
    putenv("RANDOM_STRING", random-string)
    ...run podman info
    ...now make sure that what you get from podman matches the interpolated $RANDOM_STRING
```